network-api/networkapi/wagtailpages/pagemodels/serializers.py

```diff
@@ -0,0 +1,124 @@
+import datetime
+from rest_framework import serializers
+from wagtail_airtable.serializers import AirtableSerializer
+
+from networkapi.buyersguide.fields import ExtendedYesNoField
+from networkapi.wagtailpages.pagemodels.products import TRACK_RECORD_CHOICES
+
+
+class TrackRecordChoicesSerializer(serializers.RelatedField):
+    def to_internal_value(self, data):
+        value = data.lower().strip()
+        for choice_key, choice_value in TRACK_RECORD_CHOICES:
```

Comment:
This is where we're using the track record choices. We couldn't use this property inside of the model because it's a deferred property. So I moved it outside of the model. Nothing major, but looks different from our previous code.